client/gutenberg/extensions/publicize/editor.scss

```diff
@@ -28,12 +28,12 @@ $dark-gray-500: #555d66;
 
 .jetpack-publicize-gutenberg-social-icon {
 	fill: $dark-gray-500;
-	margin-right: 0.2em;
+	margin-right: 5px;
```

Comment:
> increase the 3px of margin inside connections (to the right of the icon, and to the left of the toggle) to 5px, because 3px feels a bit too crowded

Would 4px look good too, since it's what Gutenberg grid is divided into? Not sure if the gap between icons really has anything to do with the base grid, though. :-)

https://github.com/WordPress/gutenberg/blob/0416bae17c52b0a11ec1075c0928f879264b7d75/assets/stylesheets/_variables.scss#L17-L20

Reply:
Yeah, that was the reason why I used 5px :) I'll let design folks cast their vote too, but generally I think it will look well with both 4px and 5px.